package.json

```diff
@@ -54,6 +54,7 @@
     "build": "grunt build",
     "release": "grunt release",
     "start": "node scripts/kibana --dev",
+    "debug": "node --nolazy --inspect --debug-brk scripts/kibana --no-base-path --dev",
```

Comment:
specifying `--no-base-path` in this script is going to cause weird cookie issues for people who use the defaults elsewhere isn't it? If you always run with `--no-base-path` I recommend using a bash alias to always add it locally.

Reply:
👍